tests/actions/test_core_actions.py

```diff
@@ -427,6 +432,9 @@ def test_run_integration_success(
         self,
         checkpoint_result: CheckpointResult,
     ):
+        if not os.environ.get("GX_MS_TEAMS_WEBHOOK"):
```

Comment:
I don't like programmatically skipping tests. What has happened in the past is someone makes a change and we start skipping this in CI without realizing it. I'd rather just fail and require this env variable or have this moved to a different marker where we require this.